src/Components/Web/src/Forms/InputFile/RemoteBrowserFileStreamOptions.cs

```diff
@@ -23,11 +24,12 @@ public class RemoteBrowserFileStreamOptions
 
         /// <summary>
         /// Gets or sets the maximum internal buffer size for unread data sent over a SignalR circuit.
+        /// Default -1 represents the default Pipe pauseWriterThreshold in <see cref="System.IO.Pipelines.PipeOptions" />.
         /// <para>
         /// This only has an effect when using Blazor Server.
         /// </para>
         /// </summary>
-        public int MaxBufferSize { get; set; } = 1024 * 1024;
+        public int MaxBufferSize { get; set; } = -1;
```

Comment:
Technically I guess this doesn't need to change if we're not using it any more, but doesn't matter :)